packages/fiori/src/themes/TimelineItem.css

```diff
@@ -89,6 +146,11 @@
 	pointer-events: none;
 }
 
+:host([axis-orientation="horizontal"]) .ui5-tli-bubble:focus::after {
+	top: var(--_ui5_TimelineItem_horizontal_bubble_outline_top);
```

Comment:
If you are not changing the value of the CSS variable, then you can just hard-code the value here

Reply:
Done